mlcomp/parallelm/pipeline/component_runner/java_connected_component_runner.py

```diff
@@ -52,11 +53,11 @@ def run(self, parent_data_objs):
         comp_dir = self._dag_node.comp_root_path()
         self._logger.info("comp_dir: {}".format(comp_dir))
 
-        component_jar = os.path.join(comp_dir, self._dag_node.comp_program())
-        self._logger.info("jar_file: {}".format(component_jar))
+        jar_files = glob.glob(comp_dir + "/*.jar")
+        self._logger.info("jar_files: {}".format(jar_files))
```

Comment:
Better to print a clearer message. Bear in mind that these messages will be read by our customers. So, at least for `info` severity and above, we better used clear messages.